test/resize.width.html

```diff
@@ -55,7 +55,7 @@
                     ' message back from iFrame. Expected resize'
                 )
                 ok(
-                  '300' === messageData.width,
+                  300 === messageData.width,
```

Comment:
These are the kinds of checks / changes I like.

This constitutes a breaking change, and no shenanigans around backwards-compatibility.
Old code should continue to work, but if it breaks it's because it depends on the eventual stringiness of `width` and `height`.